buildenv

```diff
@@ -8,6 +8,9 @@ GIT_VERSION="2.39.2"
 export ZOPEN_GIT_URL="https://github.com/git/git"
 export ZOPEN_GIT_DEPS="curl git make m4 perl autoconf automake help2man texinfo xz zlib openssl expat gettext coreutils diffutils bash"
 
+export ZOPEN_TARBALL_MAN_REL="2.9.5"
```

Comment:
Should this be 2.39.2?

Reply:
I don’t know why but the doc versions aren’t the same as the code versions. That’s the latest one.

Reply:
added a block comment to explain versioning

Reply:
Hmm, could it a result of the sorting?
```
git-manpages-2.39.2.tar.xz                        14-Feb-2023 17:56    544K
vs
git-manpages-2.9.5.tar.xz                          10-Aug-2017 17:58    383K
```

Reply:
fixed to use the proper level of manpages.